packages/fx-core/tests/component/generator/generator.test.ts

```diff
@@ -541,21 +541,17 @@ describe("Generator happy path", async () => {
   });
 
   it("template from fallback", async () => {
-    const foobarTemplateName = "foobarTemplate";
     const realTemplateName = "non-sso-tab";
```

Comment:
The best practice of unit test is to mock all external dependencies. You may consider when we remove `non-sso-tab` template, this test case will fail unexpectedly.

Reply:
Anyway, this test case looks much "cleaner" after the refactor. It will be better if you get rid of the dependency of a real template.

Reply:
Thanks for the feedback. I‘d replace real template with mock template.